### PR TITLE
Merge Dev to Main

### PR DESCRIPTION
**Pull request by derekc00**

### CODE REVIEW

File: Weatherly/Model/APIManager.swift [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+//  Weatherly
+//
+//  Created by admin on 6/20/22.
+//
+
+import Foundation
+
+class APIManager: ObservableObject {
+    
+  static private let apiKey = "1495b6e70be9d92ba6cafc1c60e83bd6"
```
Comment (CodeNextPaco) [inline Weatherly/Model/APIManager.swift:12]: Let's remove this. I got a complaint about it.

Comment (CodeNextPaco) [reply Weatherly/Model/APIManager.swift:12]: Or hide the API key

**Review by CodeNextPaco — CHANGES_REQUESTED** [review]

**Comment by derekc00** [comment]
I removed the key and added a line inside the readme to provide instruction on this

**Review by CodeNextPaco — APPROVED** [review]
Looks good now! Thanks

**Pull request merged**